rpc/onwire.h

```diff
@@ -95,6 +95,8 @@ struct m0_rpc_item_header2 {
 	struct m0_uint128 osr_uuid;
 	uint64_t          osr_sender_id;
 	uint64_t          osr_session_id;
+	/** @see m0_rpc_session::s_xid_list, m0_rpc_item_xid_min_update(). */
```

Comment:
Add comments to say:
The minimal active xid on sender side.

Or something similar.

Reply:
Done. Please check.